lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1832,118 +1871,145 @@ void HdVP2Mesh::_UpdateDrawItem(
         indexBuffer = const_cast<MHWRender::MIndexBuffer*>(sharedBBoxGeom.GetIndexBuffer());
     }
 
-    _delegate->GetVP2ResourceRegistry().EnqueueCommit(
-        [stateToCommit, param, primvarInfo, indexBuffer, isBBoxItem, &sharedBBoxGeom]() {
-            const HdVP2DrawItem::RenderItemData& drawItemData = stateToCommit._renderItemData;
-            MHWRender::MRenderItem*              renderItem = drawItemData._renderItem;
-            if (ARCH_UNLIKELY(!renderItem))
-                return;
-
-            MProfilingScope profilingScope(
-                HdVP2RenderDelegate::sProfilerCategory,
-                MProfiler::kColorC_L2,
-                renderItem->name().asChar(),
-                "Commit");
-
-            // If available, something changed
-            if (stateToCommit._indexBufferData)
-                indexBuffer->commit(stateToCommit._indexBufferData);
-
-            // If available, something changed
-            if (stateToCommit._shader != nullptr) {
-                renderItem->setShader(stateToCommit._shader);
-                renderItem->setTreatAsTransparent(stateToCommit._isTransparent);
-            }
+    _delegate->GetVP2ResourceRegistry().EnqueueCommit([stateToCommit,
+                                                       param,
+                                                       primvarInfo,
+                                                       indexBuffer,
+                                                       isBBoxItem,
+                                                       &sharedBBoxGeom]() {
```

Comment:
What happens here? I like the new format. Is it done automatically by clang-format and is it upgraded?

Reply:
Yes the indentation changed because of clang-format. I'm not sure why it decided to do things differently. github can ignore whitespace differences from the "gear" menu under the pull request name in the top left of the browser.